addons/mrp_subcontracting/models/stock_picking.py

```diff
@@ -36,11 +36,16 @@ def _action_done(self):
         for move in self.move_lines.filtered(lambda move: move.is_subcontract):
             # Auto set qty_producing/lot_producing_id of MO if there isn't tracked component
             # If there is tracked component, the flow use subcontracting_record_component instead
```

Comment:
You should update/delete this comment if you're going to delete the code it is related to

Reply:
done